source/manual/get-started.html.md

```diff
@@ -17,24 +17,278 @@ If you're having trouble with this guide, you can ask your colleagues or the #go
 
+## 0. Install the [Homebrew package manager](https://brew.sh) (on macOS or Linux)
+
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+```
+
 ## 1. Set up your GitHub account
 
-👉 [Set up your GitHub account](/manual/github-setup.html)
+1. Set up a [GitHub][] account. Existing personal accounts are fine to use.
+1. Ask your tech lead to add you to the [alphagov organisation][alphagov]. You will have to be added to the [GOV.UK team][govuk-team] to get access to repos & CI. Remember to click accept in the GitHub email invitation.
+1. Ask somebody with access to add your GitHub username to the [user monitoring system][user-reviewer].
+1. [Generate and register an SSH key pair][register-ssh-key] for your laptop for your GitHub account. You should use a `4096` bit key.
+1. Import the SSH key into your keychain. Once you’ve done this, you'll be able to clone repos over SSH.
+
+    ```
+    $ /usr/bin/ssh-add -K your-private-key
+    ```
+
+1. Add the above line into your `~/.bash_profile` or equivalent so that it is persistent between restarts.
+    
+1. Test that it all works by running `ssh -T git@github.com`.
+
+1. While you're here, associate your name and email to your git commits:
+
+    ```
+    $ git config --global user.email "friendly.giraffe@digital.cabinet-office.gov.uk"
+    $ git config --global user.name "Friendly Giraffe"
+    ```
+
+[GitHub]: https://www.github.com/
+[user-reviewer]: https://github.com/alphagov/govuk-user-reviewer
+[alphagov]: https://github.com/alphagov
+[govuk-team]: https://github.com/orgs/alphagov/teams/gov-uk/members
+[register-ssh-key]: https://help.github.com/articles/connecting-to-github-with-ssh/
+
+## 2. Install GDS tooling
+
+On GOV.UK we use two command-line tools day-to-day: [`govuk-connect`](https://github.com/alphagov/govuk-connect) and the [`gds-cli`](https://github.com/alphagov/gds-cli) for AWS, SSH and VPN access.
+
+To install these, run:
+
+```bash
+brew tap alphagov/gds
+brew install gds-cli govuk-connect
+```
 
-## 2. Set up govuk-docker
+The GDS CLI repository is private, so you'll need to follow the GitHub setup instructions above for the download to work.
+
+Test that both tools work by running `gds --help` and `gds govuk connect --help`.
+
+> If you see `fatal: no such path in the working tree`, it's because you're using ZSH, which has `gds` set up as a Git alias. You can either remove that alias by adding `unalias gds` to your `~/.zshrc`, or use `gds-cli` instead of `gds` for all the relevant commands.
+
+The GDS CLI requires some initial configuration:
+
+```bash
+gds config email firstname.lastname@digital.cabinet-office.gov.uk
+gds config yubikey false # If you type MFA codes from your phone
+```
+
+## 3. Set up govuk-docker
 
 We use a Docker environment - [govuk-docker][] - for local development.
 
 👉 [Learn about how we use Docker](/manual/intro-to-docker.html)
 
 👉 [Get setup with govuk-docker][govuk-docker]
```

Comment:
Maybe link straight to https://github.com/alphagov/govuk-docker/blob/master/docs/installation.md?